### PR TITLE
remove system and global headers

### PR DESCRIPTION
**Pull request by pbomta**
Reduces the size of the "uber" header file.

### CODE REVIEW

File: include/d2tmh.h [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-#define D2TMH_H
+#pragma once
 
-#include "system.h"
 #include "allegroh.h"
 #include "data.h"
 #include "definitions.h"
 #include "structs.h"
-#include "global.h"
+// #include "global.h"
```
Comment (stefanhendriks) [inline include/d2tmh.h:9]: can be removed I guess :)

**Review by stefanhendriks — APPROVED** [review]
Nice to see this. It looks like it becomes easier and easier? Although some more hard changes will come up ahead. 

A nice improvement indeed :)

File: include/d2tmh.h (hunk 1 above)
Comment (pbomta) [reply include/d2tmh.h:9]: Oops! Good catch.

**Comment by pbomta** [comment]
> Nice to see this. It looks like it becomes easier and easier? Although some more hard changes will come up ahead.
> 
> A nice improvement indeed :)

Thanks!
It's indeed slowly getting easier.

**Pull request merged**